pkg/console/operator/sync_v400.go

```diff
@@ -444,24 +444,40 @@ func (c *consoleOperator) UpdateCustomLogoSyncSource(operatorConfig *operatorsv1
 	)
 }
 
-func (c *consoleOperator) ValidateCustomLogo(operatorConfig *operatorsv1.Console) (okToMount bool, reason string, err error) {
+func (c *consoleOperator) ValidateCustomLogo(operatorConfig *operatorsv1.Console) (okToSync, okToMount bool, reason string, err error) {
 	logoConfigMapName := operatorConfig.Spec.Customization.CustomLogoFile.Name
 	logoImageKey := operatorConfig.Spec.Customization.CustomLogoFile.Key
 
 	if configmapsub.FileNameOrKeyInconsistentlySet(operatorConfig) {
 		klog.V(4).Infoln("custom logo filename or key have not been set")
-		return false, "KeyOrFilenameInvalid", customerrors.NewCustomLogoError("either custom logo filename or key have not been set")
+		return false, false, "KeyOrFilenameInvalid", customerrors.NewCustomLogoError("either custom logo filename or key have not been set")
 	}
 	// fine if nothing set, but don't mount it
 	if configmapsub.FileNameNotSet(operatorConfig) {
 		klog.V(4).Infoln("no custom logo configured")
-		return false, "", nil
+		return false, false, "", nil
+	}
+	_, err = c.configMapClient.ConfigMaps(api.OpenShiftConfigNamespace).Get(logoConfigMapName, metav1.GetOptions{})
+	// We should not mount until the configMap with the custom logo is created by user
+	if apierrors.IsNotFound(err) {
+		msg := fmt.Sprintf("source custom logo file %v in openshift-config not found", logoConfigMapName)
+		klog.V(4).Infof(msg)
+		return false, false, "FailedGetSourceConfigMap", customerrors.NewCustomLogoError(msg)
```

Comment:
Nit, you use `FailedGetSource` as the `Type` will be `CustomLogoSync`. so:

`CustomLogoSync` -> `FailedGetSource` will be what is seen in monitoring.